Makefile

```
@@ -27,8 +28,7 @@ catalog-csv-metadata/rhacs-operator/catalog.json: catalog-template.yaml $(OPM)
 	$(OPM) alpha render-template basic --migrate-level bundle-object-to-csv-metadata $< > $@
 
 # update template/catalog-template.yaml based on bundles.yaml file.
-.PHONY: catalog-template.yaml
-catalog-template.yaml: bundles.yaml
+catalog-template.yaml: bundles.yaml $(wildcard cmd/**/*.go)
```

Isn't it only `cmd/generate-catalog`?

```suggestion
catalog-template.yaml: bundles.yaml $(wildcard cmd/generate-catalog/*.go)
```

Also, I suggest introducing a variable for `./cmd/generate-catalog/` so that both task and go run command use the same thing.